src/deviceApiCalls/schemas/storeUserData.params.json

```
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "StoreUserDataParams",
+    "type": "object",
+    "description": "Used to store Email Protection auth credentials.",
+    "additionalProperties": false,
+    "properties": {
+      "token": {
+        "type": "string"
+      },
+      "username": {
```

From my previous comment -- I think this should be:
```suggestion
      "userName": {
```